frontend/src/views/supp-allowances/SupplementaryAllowanceView.vue

```diff
@@ -140,15 +140,15 @@ export default {
         // navigate from the Application Confirmation page (after the providers have just submitted their core funding application)
         if (this.$route.params.applicationGuid) {
           this.application = await ApplicationService.getApplication(this.$route.params.applicationGuid)
-          this.application.fundingAgreement = await FundingAgreementService.getActiveFundingAgreementByApplicationId(this.$route.params.applicationGuid)
+          this.application.fundingAgreement = await FundingAgreementService.getActiveFundingAgreementByApplicationId(this.$route.params.applicationGuid, true)
```

Comment:
should we add "true" flag to getActiveFundingAgreementByApplicationId in ApplicationsHistoryView? it's for enabling the Supplementary Application button

Reply:
no we don't need to - the check that happens in ApplicationService looks at the endDate to confirm if the FA (or MOD) is active. End dates do not change across FA and MOD's . 

We re-load the FA from Dynamics on the SuppApp page anyways to get the original FA so I think we are okay here.